test/parallel/test-fs-realpath.js

```diff
@@ -225,7 +225,7 @@ function test_cyclic_link_protection(callback) {
   });
   assert.throws(() => {
     fs.realpathSync(entry);
-  }, common.expectsError({ code: 'ELOOP', type: Error }));
+  }, /^Error: ELOOP: too many symbolic links encountered, stat/);
```

Comment:
You asked that "regular" errors checks won't use `common.expectsError`.
I'll check again.